doc/classes/TileSet.xml

```diff
@@ -45,7 +45,7 @@
 			</argument>
 			<description>
 				Determines when the auto-tiler should consider two different auto-tile IDs to be bound together.
-				Also, neighbor_id can (and will) be -1 when checking a tile against air.
+				Also, [code]neighbor_id[/code] can (and will) be [code]-1[/code] when checking a tile against air.
```

Comment:
Simplify "can (and will) be" to "will be".

But it's a bit unclear to me what "checking a tile against air" means. "air" is not really a TileMap/TileSet concept. Do you mean an empty/invalid tile? `-1` is actually `[constant TileMap.INVALID_CELL]`, so it might be good to reference it instead.

So I would suggest:
```suggestion
				[b]Note:[/b] [code]neighbor_id[/code] will be [code]-1[/code] ([constant TileMap.INVALID_CELL]) when checking a tile against an empty neighbor tile.
```